meerk40t/gui/wxmribbon.py

```diff
@@ -103,14 +110,113 @@ def GetBestSize(self):
                     ht += 10
                     if self.GetLabel() != "":
                         ht += 23
-                    # print ("Found: wd=%s, ht=%s" % (wd, ht))
-                    size = wx.Size(wd, ht)
+                    else:
+                        ht += 10
                     break
+        else:
+            wd = size[0]
+            ht = size[1]
```

Comment:
If bar size[0]<50 and bar has no children `wd` and `ht` never get set and crashes.

Reply:
Okay will fixit